Client/core/Graphics/CGraphics.cpp

```diff
@@ -158,46 +158,47 @@ void CGraphics::DrawString(int iX, int iY, unsigned long dwColor, float fScale,
 // Slow
 void CGraphics::DrawStringOutline(const RECT& rect, unsigned long ulColor, const wchar_t* szText, unsigned long ulFormat, LPD3DXFONT pDXFont)
 {
-    const uint   uiKernelSizeX = 5;
-    const uint   uiKernelSizeY = 5;
-    const float* pKernel;
+    constexpr int iKernelSizeX = 5;
+    constexpr int iKernelSizeY = 5;
```

Comment:
why has the type changed from uint to int?

Reply:
it was unnecessary for it to be `uint`, it was converted to int anyways.

Reply:
I see, so iKernelSizeX is eventually used to create a RECT, which uses `long`, which is essentially just a (signed) int. Okay 👍

Reply:
Exactly. The compiler probably noticed it already, but i was like why not to change it.